web/src/pages/Cases/CaseDetails/Overview/index.tsx

```diff
@@ -0,0 +1,87 @@
+import React from "react";
+import styled from "styled-components";
+import { useParams } from "react-router-dom";
+import { formatEther } from "viem";
+import { useDisputeDetailsQuery } from "queries/useDisputeDetailsQuery";
+import { useDisputeTemplate } from "queries/useDisputeTemplate";
+import { useCourtPolicy } from "queries/useCourtPolicy";
+import { Periods } from "consts/periods";
+
+import DisputeInfo from "components/DisputeCard/DisputeInfo";
+import Verdict from "components/Verdict/index";
+import { useVotingHistory } from "hooks/queries/useVotingHistory";
+import { getLocalRounds } from "utils/getLocalRounds";
+import { DisputeBasicInfo } from "./DisputeBasicInfo";
+import { Policies } from "./Policies";
+
+const Container = styled.div`
+  width: 100%;
+  height: auto;
+  display: flex;
+  flex-direction: column;
+  gap: calc(16px + (32 - 16) * (min(max(100vw, 375px), 1250px) - 375px) / 875);
+  padding: calc(16px + (32 - 16) * (min(max(100vw, 375px), 1250px) - 375px) / 875);
+
+  > h1 {
+    margin: 0;
+  }
+
+  > hr {
+    width: 100%;
+  }
+`;
+
+const Divider = styled.hr`
+  display: flex;
+  border: none;
+  height: 1px;
+  background-color: ${({ theme }) => theme.stroke};
+  margin: 0;
+`;
+
+interface IOverview {
+  arbitrable?: `0x${string}`;
+  courtID?: string;
+  currentPeriodIndex: number;
+}
+
+const Overview: React.FC<IOverview> = ({ arbitrable, courtID, currentPeriodIndex }) => {
+  const { id } = useParams();
+  const { data: disputeTemplate } = useDisputeTemplate(id, arbitrable);
+  const { data: disputeDetails } = useDisputeDetailsQuery(id);
+  const { data: courtPolicy } = useCourtPolicy(courtID);
+  const { data: votingHistory } = useVotingHistory(id);
+  const localRounds = getLocalRounds(votingHistory?.dispute?.disputeKitDispute);
+  const courtName = courtPolicy?.name;
+  const court = disputeDetails?.dispute?.court;
+  const rewards = court ? `≥ ${formatEther(court.feeForJuror)} ETH` : undefined;
+  const category = disputeTemplate?.category ?? undefined;
+
+  return (
+    <>
+      <Container>
+        <DisputeBasicInfo disputeTemplate={disputeTemplate} />
```

Comment:
maybe change the name of this component so that it doesn't get mixed up with `DisputeInfo` 🤔 something like `DisputeQuestion` or `DisputeContext` seems fine I think 🤔